resources/assets/lib/beatmap-discussions/editor-discussion-component.tsx

```diff
@@ -35,6 +38,7 @@ interface Props extends RenderElementProps {
   readOnly?: boolean;
 }
 
+@observer
```

Comment:
looks like this isn't actually needed because `this.nearbyDiscussions` is called by child component `nearbyIndicator` as part of the `DraftsContext.Consumer` (and apparently lost the reactive context from this).

Alternatively, keeping this `@observer` but removing `@action` and `makeObservable`, and then calling `this.nearbyDiscussions()` right before `const nearbyIndicator =` also removes the warning as the cache would then be filled within reactive context (this component being observer) and the subsequent `nearbyDiscussions` will use the cached value.

Reply:
better would be to make the whole thing observable and use `@computed` but I'm not fixing anything using slate without #9062 👀

Reply:
oh yeah, I forgot about that one 🫠